cypress/support/commands.js

```diff
@@ -24,6 +24,58 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
+// Calls action in the Nuxt store
+Cypress.Commands.add("dispatchToNuxtStore", (p_action, p_data) => {
```

Comment:
`dispatchToNuxtStore`,`loadDataDictionary`, and `loadDataTable` are implemented for clarity in commonly reused test code.